localstack/ext/java/src/main/java/cloud/localstack/Localstack.java

```diff
@@ -145,7 +156,17 @@ private static void ensureInstallation() {
             LOG.info(logMsg);
             messagePrinted = true;
             deleteDirectory(dir);
-            exec("git clone " + LOCALSTACK_REPO_URL + " " + TMP_INSTALL_DIR);
+            if(devEnvironment) {
+                Path localstackDir = Paths.get("../../..");
+                Path tempLocalstackDir = Paths.get(TMP_INSTALL_DIR);
+                try {
+                    copyFolder(localstackDir, tempLocalstackDir);
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
Things are likely going to fail downstream if we hit an exception here. Let's better use a fail-fast approach - can we reraise a `new RuntimeException(e)` here?